### PR TITLE
Removed Duplicate Internships

### PR DESCRIPTION
**Pull request by SadafKausar2025**
fixed issue : #294 

There where many duplicate internships available in different pages I removed them and kept only one of them

1. Hactoberfest -- Repeated 3 times
2. season of KDE -- Repeated 2 times
3. Google summer of Docs -- Repeated 2 times
4. free software foundation -- Repeated 2 times
5. MLH Felloship         -- Repeated 2 times

I kept 1 One them and removed the duplicate ones

### CODE REVIEW

**Comment by SadafKausar2025** [comment]
hello @HimanshuNarware 
can you please review this PR?

**Comment by SadafKausar2025** [comment]
hello @HimanshuNarware 
please review this PR and merge it

File: src/Component/Navbar.jsx [hunk 1]
```diff
@@ -29,83 +29,133 @@ function Navbar() {
...
-            <Link to="/" className='link'>Home</Link>
+            <Link to="/" className="link">
+              {" "}
+              <i class="fa-solid fa-house"></i> Home
+            </Link>
           </div>
           <div className="cart-item">
-            <Link to="/intern" className='link'> Internships</Link>
+            <Link to="/intern" className="link">
+              {" "}
```
Comment (HimanshuNarware) [inline src/Component/Navbar.jsx:53]: why you have added an icon on the navbar

**Comment by HimanshuNarware** [comment]
@SadafKausar2025  I think this issue is only assigned for removing duplicate internships, don't go on the navbar, we don't need icons on the navbar

**Comment by SadafKausar2025** [comment]
Ohh sorry for that 
I am removing it

**Comment by SadafKausar2025** [comment]
hello @HimanshuNarware 
Yes I have removed all icons from Navbar .
It happens by mistake . sorry for this.

Now it is all ok

Screenshot (1973)

File: src/Component/Navbar.jsx (hunk 1 above)
Comment (SadafKausar2025) [reply src/Component/Navbar.jsx:53]: sorry , I have edited icons very before and forget to remove. That's why this happend

Comment (HimanshuNarware) [reply src/Component/Navbar.jsx:53]: ok

**Comment by SadafKausar2025** [comment]
hello @HimanshuNarware 
please give levels also

**Pull request merged**